farmerbot/manager/rmb.go

```diff
@@ -0,0 +1,67 @@
+package manager
+
+import (
+	"context"
+	"time"
+
+	"github.com/threefoldtech/tfgrid-sdk-go/farmerbot/constants"
+	"github.com/threefoldtech/tfgrid-sdk-go/farmerbot/models"
+	"github.com/threefoldtech/tfgrid-sdk-go/rmb-sdk-go"
+	"github.com/threefoldtech/zos/pkg"
+)
+
+// RMB is an rmb abstract client interface.
+type RMB interface {
+	SystemVersion(ctx context.Context, nodeTwin uint32) error
+	Statistics(ctx context.Context, nodeTwin uint32) (stats models.ZosResourcesStatistics, err error)
+	GetStoragePools(ctx context.Context, nodeTwin uint32) (pools []pkg.PoolMetrics, err error)
+	ListGPUs(ctx context.Context, nodeTwin uint32) (gpus []models.GPU, err error)
+}
+
+type rmbNodeClient struct {
+	rmb        rmb.Client
+	rmbTimeout time.Duration
+}
+
+func NewRmbNodeClient(rmb rmb.Client) rmbNodeClient {
```

Comment:
Public method return a private object.  
```suggestion
func NewRmbNodeClient(rmb rmb.Client) *RmbNodeClient {
```

> See how i also returned a pointer not a value, because it's `*RmbNodeClient` what implementes RMB not `RmbNodeClient`